### DISCUSSION TITLE
Query regarding homework 9

### DISCUSSION THREAD

**Discussion by gagankhetwani** [cafawo/FinancialDataAnalytics#25] [Q&A] [answered]
Hello,

In task 1 of HW 9, I can see the output of my location map in codespace but I cannot see the output as the map in the GitHub repository file. Please refer to the screenshot below
![GitHub](https://github.com/iwh-halle/FinancialDataAnalytics/assets/74928103/3cb298f9-19fc-4376-8fc5-8f944a2434a3)
GitHub Screenshot

**Comment by gagankhetwani** [comment]
![Codespace](https://github.com/iwh-halle/FinancialDataAnalytics/assets/74928103/afa7de17-efa2-4465-b11d-ebf5bf14e673)
Codescape screenshot

**Comment by julian-456** [comment, marked as answer]
That should not be a problem. Github doesn't always render all outputs. 
In particular, dynamic content will probably not be displayed.

**Reply by cafawo** [reply]
Agreed. As long as it runs, you're good.
